static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -35,6 +39,29 @@ define([
                     bean.on(el, 'click', askQuestion);
```

Comment:
I think it needs to be on each question in order to know which question it was that was clicked on.